pool/routes/routes.ml

```diff
@@ -36,12 +36,12 @@ module Public = struct
         ~middlewares:
           [ CustomMiddleware.Tenant.valid_tenant ()
           ; CustomMiddleware.Context.context ()
+          ; CustomMiddleware.Logger.logger
```

Comment:
Should we not have it on the global middlewares?

Reply:
It's not going to be useful because we need the context, so it has to come before the context middleware